src/text/parsers/text_support.rs

```diff
@@ -39,6 +42,16 @@ pub(crate) fn escaped_char(input: &str) -> IResult<&str, StringFragment> {
     )(input)
 }
 
+/// Matches an escaped literal (like '\n') or a hex escape (like `\x`), returning the appropriate
+/// substitute character as a [StringFragment::EscapedChar]. Does NOT match Unicode escapes
+/// ('\u' or '\U').
+pub(crate) fn escaped_char_no_unicode(input: &str) -> IResult<&str, StringFragment> {
+    map(
+        preceded(char('\\'), alt((escaped_hex_char, escaped_char_literal))),
+        StringFragment::EscapedChar,
+    )(input)
+}
```

Comment:
🗺️ Clobs aren't supposed to support Unicode escapes, so they can use this function instead of `escaped_char_unicode` below.